packages/docs/src/lang/en/components/DataTables.json

```diff
@@ -117,6 +117,7 @@
       "headersLength": "Can be used in combination with `hide-default-header` to specify the number of columns in the table to allow expansion rows and loading bar to function properly",
       "height": "Set an explicit height of table",
       "hideDefaultHeader": "Hide the default headers",
+      "itemClass": "Function that takes item as argument and returns the list of classes to add to the row of the item",
```

Comment:
```suggestion
      "itemClass": "Function that takes item as argument and returns the list of classes to add to the item",
```

Reply:
Not sure, but this text may be better. @vuetifyjs/core-team thoughts?

Reply:
"Function that takes item as argument and returns the list of classes to add to table row corresponding to the item" maybe?

Reply:
Ah, I was wrong.